_layouts/default.html

```diff
@@ -9,6 +9,7 @@
   {% include head.html %}
 
   <body>
+    {% include gtm_iframe.html %}
```

Comment:
@konklone this is not an attempt to scale out into many gov sites yet. This is an independent, DAP team only GTM account. Other agencies are currently also using their own GTM accounts to deploy the DAP code. This was mainly put in place so the DAP team had an account on a live site with which to experiment adding custom events, etc. 

That said, I'm not sure how important it is to have it right after `<body>`. As you noted, I was following the recommendations.

![screen shot 2017-02-24 at 10 11 35 am](https://cloud.githubusercontent.com/assets/6556888/23308617/b7ca97ca-fa79-11e6-88da-17832a5e0b0c.png)